Gundi/tests/Gundi.Tests/Generated/Gundi/Gundi.UnionGenerator/CSharpUnion.g.cs

```diff
@@ -0,0 +1,246 @@
+﻿#nullable enable 
+
+using System;
```

Comment:
Is it necessary to commit generated code ?

Reply:
According to https://andrewlock.net/creating-a-source-generator-part-6-saving-source-generator-output-in-source-control/, I think we can track those files. Whenever we change something in the generator, we will be able to determine what has changed.